### ISSUE TITLE
How to detect as an unknown face?

### ISSUE THREAD

**Issue by vikramreddy0307** [abdulfatir/prototypical-networks-tensorflow#10]
Hello,Thank you for sharing your work in detail.Could you please help me how to  identify unknown face in query images as we don't have threshold in softmax outputs?.

Thanks in advance
